src/gridtools/clang/verify.hpp

```diff
@@ -109,7 +109,7 @@ class verifier {
 
     // Check dimensions
     auto check_dim = [&](uint_t dim1, uint_t dim2, uint_t size, const char* dimstr) {
-      if(dim1 != dim2 || dim1 < size) {
+      if(dim1 != dim2 || (dim1 > 1 && dim1 < size)) {
```

Comment:
I think dim < size should not trigger an error. We might have  a storage with size 80 but only want to check 10 levels. Isnt?